src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -54,24 +54,48 @@ public class CameraData
         private readonly Vector3D defaultCameraUpDirection = new Vector3D(0, 1, 0);
         private const double defaultNearPlaneDistance = 0.1;
         private const double defaultFarPlaneDistance = 10000000;
-         
+
+        [JsonIgnore]
         public Point3D EyePosition { get; set; }
```

Comment:
it seems kind of backward to have these properties with setters - perhaps these should be getters only, that construct the appropriate types when clients ask for them from the other properties you have added. This implementation lets a camera have 2 eye positions, 2 lookDirections and 2 upDirections that can get out of sync.